Lib/logging/__init__.py

```diff
@@ -252,6 +251,18 @@ def _releaseLock():
         _lock.release()
 
 
+def _get_lock() -> ContextManager:
+    """
+    Get the module-level lock using a context manager for serializing access to
+    shared data and safely releasing the lock once finished.
+    """
+    if _lock:
```

Comment:
Since 2017, _lock is no longer None by default: commit a6a4dc816d68df04a7d592e0b6af8c7ecc4d4344.

Maybe it's time to remove ``if _lock:`` here and in _acquireLock().

IMO _get_lock() is useless, you should just write ``with _lock:``.

Reply:
Makes perfect sense, that's a lot simpler. Done.